airmozilla/starred/templates/starred/home.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'main/home.html' %}
+{% set page = page|default('home') -%}
+
+{% block content_main %}
+  {% for event in events %}
+  {{ article(event) }}
+  {% endfor %}
+  <!-- OR
+  {% for star in starred %}
+  {{ article(star.event) }}
+  {% endfor %}
```

Comment:
Yes! This is much better. That way we don't need to make a list of events in the view function.